react/__tests__/components/ProductPrice.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ProductPrice from './../../ProductPrice'
+import { render } from 'enzyme'
```

Comment:
I would rather use `react-testing-library`.

It makes it easier to avoid testing implementation details.

Reply:
I wrote some docs on how to setup:
https://docs.google.com/document/d/1RRjBWTtA9tO_6ci29XUbtPCR5_QkpcGM6q_VRUwstuc/edit?usp=sharing